app/containers/Forms/FormArrayFieldTemplate.tsx

```diff
@@ -16,11 +16,32 @@ const FormArrayFieldTemplate: React.FunctionComponent<ArrayFieldTemplateProps> =
         return (
           <div key={element.index}>
             <Form.Row>
-              <Col xs={12} md={element.hasRemove ? 11 : 12}>
-                {element.children}
-              </Col>
+              {uiSchema.linkProduct ? (
```

Comment:
The FormArrayFieldTemplate is generic, and should not have any logic tied to a specific form. We can look into that on Monday to figure out how to have the same results without changing this file